account_invoice_production_lot/demo/sale.xml

```diff
@@ -0,0 +1,74 @@
+        <field name="property_stock_inventory" ref="location_opening"/>
+        <field name="categ_id" ref="product.product_category_1"/>
+        <field name="standard_price">70.0</field>
+        <field name="list_price">100.0</field>
+        <field name="type">product</field>
+        <field name="cost_method">average</field>
+        <field name="valuation">real_time</field>
+        <field name="description">Ice cream can be mass-produced and thus is widely available in developed parts of the world. Ice cream can be purchased in large cartons (vats and squrounds) from supermarkets and grocery stores, in smaller quantities from ice cream shops, convenience stores, and milk bars, and in individual servings from small carts or vans at public events.</field>
+        <field name="default_code">001</field>
+    </record>
+    <record id="lot_icecream_0" model="stock.production.lot">
+        <field name="name">Lot0 for Ice cream</field>
+        <field name="product_id" ref="product_icecream_b"/>
+    </record>
+    <record id="lot_icecream_1" model="stock.production.lot">
+        <field name="name">Lot1 for Ice cream</field>
+        <field name="product_id" ref="product_icecream_b"/>
+    </record>
+    <record id="sale_order_0" model="sale.order">
+        <field name="partner_id" ref="base.res_partner_2"/>
+        <field name="partner_invoice_id" ref="base.res_partner_2"/>
+        <field name="partner_shipping_id" ref="base.res_partner_2"/>
+        <field name="user_id" ref="base.user_admin"/>
+        <field name="date_order" eval="(DateTime.today() - relativedelta(months=1)).strftime('%Y-%m-%d %H:%M')"/>
+    </record>
+    <record id="sale_order_line_0" model="sale.order.line">
+        <field name="order_id" ref="sale_order_0"/>
+        <field name="name">Ice Cream B</field>
+        <field name="product_id" ref="product_icecream_b"/>
+        <field name="product_uom_qty">1.0</field>
+        <field name="product_uom" ref="uom.product_uom_unit"/>
+        <field name="price_unit">100.00</field>
+    </record>
+    <record id="sale_order_1" model="sale.order">
+        <field name="partner_id" ref="base.res_partner_4"/>
+        <field name="partner_invoice_id" ref="base.res_partner_address_13"/>
+        <field name="partner_shipping_id" ref="base.res_partner_address_13"/>
+        <field name="user_id" ref="base.user_admin"/>
+        <field name="date_order" eval="(DateTime.today() - relativedelta(months=1)).strftime('%Y-%m-%d %H:%M')"/>
+    </record>
+    <record id="sale_order_line_1" model="sale.order.line">
+        <field name="order_id" ref="sale_order_1"/>
+        <field name="name">Ice Cream B</field>
+        <field name="product_id" ref="product_icecream_b"/>
+        <field name="product_uom_qty">2.0</field>
+        <field name="product_uom" ref="uom.product_uom_unit"/>
+        <field name="price_unit">100.00</field>
+    </record>
+    <record id="sale_order_2" model="sale.order">
```

Comment:
I think that if we want to showcase an invoice with lot, we should add a picking and an invoice with proper links between them and mention in the README that with demo data, invoice number X is a sample of an invoice with lots.